### DISCUSSION TITLE
Why the #(...) syntax for tuples?

### DISCUSSION THREAD

**Discussion by pwtail** [gleam-lang/gleam#2075] [Questions & support] [answered]
Mind if I ask: why the `#` syntax for tuples? If the parentheses are not taken?
`("this", "tuples")` look prettier.

**Comment by lpil** [comment, marked as answer]
Because without some kind of proceeding token it would be ambiguous with function calls. The alternative would be to require `;` at the end of lines, as Rust, Java, etc do.
